zellij-utils/src/input/layout.rs

```diff
@@ -29,14 +33,23 @@ pub enum SplitSize {
     Fixed(u16),  // An absolute number of columns or rows
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(crate = "self::serde")]
+pub enum Run {
+    #[serde(rename = "plugin")]
+    Plugin(Option<PathBuf>), // 1 to 100
+    #[serde(rename = "command")]
+    Command(RunCommand), // An absolute number of columns or rows
+}
```

Comment:
I think maybe the comments here are not relevant?

Reply:
Oh,good catch!